inception-project/inception
# #3877 - Show a bit of context in the annotation sidebar

**What's in the PR**
- Display part of the text after the covered text as muted text
- No smart cut-off
- Trailing text may cause annotation item to wrap and thereby take up more vertical space

**How to test manually**
* Create a few annotations and check what they look like in the annotation sidebar

**Automatic testing**

**Documentation**

**Demo**
![by-position](https://user-images.githubusercontent.com/1410238/225423825-7a1a9a38-2b09-4b69-96f5-98b088a4ecd7.png)

![by-label](https://user-images.githubusercontent.com/1410238/225423849-f25c4fc5-2598-40ae-9cdd-948d18c52a5e.png)
